docs/integrations/systemd.rst

```diff
@@ -21,6 +27,9 @@ Then create the systemd service file in ``/etc/systemd/system/postgrest.service`
   After=postgresql.service
 
   [Service]
+  User=postgrest
+  Group=postgrest
+  Restart=on-failure:3
```

Comment:
@haowuge Hm, what does the 3 here mean? Could you link the docs for that?

I cannot find a reference to it on https://www.freedesktop.org/software/systemd/man/systemd.service.html#Restart=

Reply:
@steve-chavez Thank you for your guidance,
I mistakenly remembered 'Restart=on failure: 3', which is actually a parameter of the docker. 
Sorry

[Docker supports the following restart policies: ](https://docs.docker.com/engine/reference/run/)